src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -237,6 +270,39 @@ export const WorkspaceDashboard = _.flow(
             storageCostEstimate
           ])
         ]),
+        div({ style: styles.header }, ['Tags']),
+        div({ style: { marginBottom: '0.5rem' } }, [
+          allTags && h(CreatableSelect, {
+            isClearable: true,
+            isSearchable: true,
+            value: newTag,
+            placeholder: 'Add a tag',
+            onChange: data => this.addTag(data.value),
+            styles: { container: base => ({ ...base, wordWrap: 'break-word' }) },
+            options: _.map(value => {
+              return {
+                value: value.tag,
+                label: `${value.tag} (${value.count})`
+              }
+            }, allTags)
```

Comment:
With a large number of tags, this can cause the browser to slow down or crash. We need to rethink the approach here, and limit the number of tags we try to display at once.